### DISCUSSION TITLE
Table header label is uppercase for no reason (bug?)

### DISCUSSION THREAD

**Discussion by MichielVervaart** [filamentphp/filament#11637] [Help]
### Package

Table builder

### Package Version

v3.2.41

### How can we help you? 

The labels in my tables are uppercased when sortable isn't enabled. I cant find the reason why because the html being rendered looks fine but is not. 

![image](https://github.com/filamentphp/filament/assets/49039495/b791403f-71dd-4df8-9f22-3b97607a6e6a)

![image](https://github.com/filamentphp/filament/assets/49039495/68108179-c3be-4466-9af5-f1b304423f53)

Could someone tell me what im doing wrong? In the html inspector the label is showing as expected and i cant find any css classes that might cause the html to get uppercased.

![image](https://github.com/filamentphp/filament/assets/49039495/8da60f8f-0a90-4feb-87bb-9f3ad45e68cb)

**Comment by leandrocfe** [comment]
Are you using custom css?

**Reply by MichielVervaart** [reply]
Nope its just a default table.

**Reply by leandrocfe** [reply]
Are you using the panel builder or only the table builder?

**Reply by MichielVervaart** [reply]
Just the tablebuilder :)
